### ISSUE TITLE
Features presented are impacted by model “type” 

### ISSUE THREAD

**Issue by MIKE-DAILEY** [FrSky-OS/discussion#9]
For those new to the simulator it is important to know that the “type” of model selected seems to impact which features are populated or available on the screens. 

Mike

**Comment by AloftHobbies** [comment]
Yep - See issue one for ways to improve the functionality of this, and I'll pull a new issue for a model wizard that I think would be very good to have.
https://github.com/FrSky-OS/discussion/issues/1
